apps/docs/docs/user/intro/intro.md

```diff
@@ -1,10 +1,13 @@
 ---
 sidebar_position: 1
+title: Getting Started
 ---
 
 # Introduction to Jayvee
 
-Jayvee is a domain-specific language (DSL) for data engineering - the cleaning and preprocessing of data for later activities like data science or machine learning. You can use Jayvee to **model an ETL pipeline** and the command-line interpreter to **run the ETL pipeline** on your local machine. 
+Jayvee is a domain-specific language (DSL) for automated processing of data pipelines.
+The Jayvee interpreter allows executing such data pipelines on local machines.
+Data engineers can use Jayvee and its interpreter to clean and preprocess data for later activities like data science or machine learning.
```

Comment:
What are these changes about? If they are from you, I'd revert them just to keep this focused.

Reply:
These changes were conflicts with the main branch and needed to be resolved.